tokio/tests/rt_metrics.rs

```diff
@@ -105,7 +105,7 @@ fn worker_park_count() {
         time::sleep(Duration::from_millis(1)).await;
     });
     drop(rt);
-    assert!(2 <= metrics.worker_park_count(0));
+    assert!(1 <= metrics.worker_park_count(0));
```

Comment:
There really should only be 1 park in this test, however, there often are more due to #5207. I believe the original test author just included the number they observed when implementing metrics.